docker-compose.yml

```diff
@@ -459,6 +459,49 @@ services:
             - prosody
         networks:
             meet.jitsi:
+    # Log Analyser
```

Comment:
Please keep the whole stack in a separate yaml file and not part of the standard setup.